src/config/src/meta/stream.rs

```diff
@@ -594,6 +594,8 @@ pub struct UpdateStreamSettings {
     pub index_all_values: Option<bool>,
     #[serde(default)]
     pub pattern_associations: UpdateSettingsWrapper<PatternAssociation>,
+    #[serde(default)]
+    pub disable_distinct_fields: Option<bool>,
```

Comment:
Do `None` and `Some(false)` have different meanings here? If not, this would be better off being just `bool`.